### PR TITLE
[devtools-translate] Spanish - What's New in DevTools (Chrome 107)

### LINKED ISSUE #3755: [devtools-translate] Spanish - What's New in DevTools (Chrome 107)
Post: https://developer.chrome.com/blog/new-in-devtools-107/
Translated by: Oct-11

Translator: @midudev
Reviewers: @Caballerog, @and-oli

### PR DESCRIPTION
**Pull request by midudev**
Fixes #3755

Changes proposed in this pull request:

- Chrome 107 what's new in Spanish

### CODE REVIEW

**Comment by jecfish** [comment]
Thanks Miguel! Over to @Caballerog and @and-oli for reviews!

**Review by and-oli — COMMENTED** [review]
Excelente!

LGTM % typo.

Muchas gracias Midu 👍

**Comment by Caballerog** [comment]
Genial! 

LGTM

File: site/es/blog/new-in-devtools-107/index.md [hunk 1]
```diff
@@ -4,63 +4,63 @@ title: 'Qué hay de nuevo en DevTools (Chrome 107)'
...
 
-{% Partial 'devtools/banner.md' %}
-
-<!-- Translation instructions:
-  1. Remove the "draft: true" tag above when submitting PR
-  2. Provide translations under each of the English commented original content
-  3. Translate the "description" tag above
-  4. Translate all the <img> alt text
-  5. Update the whats-new.md file -->
+{% include 'partials/devtools/es/banner.md' %}
```
Comment (jecfish) [inline site/es/blog/new-in-devtools-107/index.md:18]:
```suggestion
{% Partial 'devtools/banner.md' %}

{% YouTube id='1uwv6HbR8HU' %}
```

**Review by jecfish — APPROVED** [review]

**Pull request merged**